Source/missiles.cpp

```diff
@@ -1754,6 +1754,12 @@ void AddPhasing(Missile &missile, AddMissileParameter &parameter)
 		return;
 	}
 
+	if (&player != MyPlayer) {
+		if (!PosOkPlayer(player, missile.position.tile))

```

Comment:
> ... since that remote client has already assigned a destination result and sent that with the packet information.

What packet information are you talking about? The spell doesn't compute a destination until the hit frame, and no packets get sent by `ProcessTeleport()` that I can see.

At this point, `missile.position.tile` is the location of the player who cast it. What you are relying on here is not a destination that was sent by the remote client, but rather the sync packet that updates the player's information after the spell resolves. If that is your intent, you may as well just unconditionally delete the Phasing missile.

Reply:
Thanks for clarifying. I'll try another way to achieve the desired change. Removing the phasing missile might be the way to go. We could pre-calculate a random position and utilize Teleport's code